.buildkite/pipeline.yml

```diff
@@ -410,6 +410,14 @@ steps:
           slurm_mem: 20GB
           slurm_gpus: 1
 
+      - label: "GPU Slabplanet: extra atmos diagnostics"
+        key: "gpu_slabplanet_atmos_diags"
```

Comment:
Could be include the debug plot output in this run (or preferably in all runs as a default) at the end of the simulation? I think we need to modify 
```
    comms_ctx.device == ClimaComms.CPUSingleThreaded() &&
        comms_ctx isa ClimaComms.SingletonCommsContext &&
        debug(cs, joinpath(dir_paths.artifacts, "endstates_"))
```
at the end of the driver.